package.json

```
@@ -25,13 +25,15 @@
     "prepack": "yarn build"
   },
   "dependencies": {
-    "@jupiterone/integration-sdk": "^3.1.0",
+    "@jupiterone/integration-sdk-core": "^1.1.0",
     "@lifeomic/attempt": "^3.0.0",
     "meraki": "^1.5.0",
     "node-fetch": "^2.6.0",
     "promisfy": "^1.2.0"
   },
   "devDependencies": {
+    "@jupiterone/integration-sdk-cli": "^1.1.0",
+    "@jupiterone/integration-sdk-testing": "^1.1.0",
```

```suggestion
    "@jupiterone/integration-sdk-dev-tools": "^1.1.0",
    "@jupiterone/integration-sdk-testing": "^1.1.0",
```
Thanks to https://github.com/JupiterOne/sdk/blob/master/packages/integration-sdk-dev-tools/package.json.

---

Oh, though it will require some other changes to get the full benefit: https://github.com/JupiterOne/sdk/blob/master/packages/integration-sdk-dev-tools/README.md